modules/oc-mirror-v2-about.adoc

```diff
@@ -13,18 +13,37 @@ To use the new Technology Preview version of oc-mirror, add the `--v2` flag to t
 
 oc-mirror plugin v2 has the following features:
 
-* Verifies that the complete image set specified in the image set config is mirrored to the mirrored registry, regardless of whether the images were previously mirrored or not.
+* Verifies that the complete image set specified in the image set configuration file is mirrored to the mirrored registry, regardless of whether the images were previously mirrored or not.
 
-* Uses a cache system instead of metadata.
+* Uses a cache system instead of metadata, which prevents the need to start the mirroring process over in the case of a failure for a single step of the process.
+// Does the cache about refer to the cache of files mentioned in Luigi's demo of the delete feature, where he says that the plugin uses a local cache to cache images in case of a failure? If so, I can expand this bullet point to explicitly talk about how the cache keeps images around in case of failure, preventing the need to start over.
 
 * Maintains minimal archive sizes by incorporating only new images into the archive.
 
 * Generates mirroring archives with content selected by mirroring date.
 
-* Can generate `ImageDigestMirrorSet` (IDMS), `ImageTagMirrorSet` (ITMS), instead of `ImageContentSourcePolicy` (ICSP) for the full image set, rather than just for the incremental changes.
+* Can generate `ImageDigestMirrorSet` (IDMS) and `ImageTagMirrorSet` (ITMS) resources, which cover the full image set, instead of an `ImageContentSourcePolicy` (ICSP) resource, which only covered incremental changes to the image set for each mirroring operation.
```

Comment:
should we specify like `which only covered incremental changes to the image set for each mirroring operation during v1` ?

Reply:
If there's no way to generate ICSP with v2, then I think we can clarify that difference here, sure. If you can technically generate ICSP with v2, it might make more sense not to call out v1 here.

Reply:
we do not generate ICSP with v2, with v2 it is only ITMS, IDMS & catalogSources.